db/view/view.cc

```diff
@@ -1799,6 +1799,60 @@ static endpoints_to_update get_view_natural_endpoint_vnodes(
     return {.natural_endpoint = view_endpoints[idx].get().host_id()};
 }
 
+static std::optional<locator::host_id> get_unpaired_view_endpoint(
```

Comment:
It's not an obvious what is an "unpaired view endpoint", why there is always zero or one of these and not more. I think a comment about the purpose of this function would have helped. There are some helpful comments below, but I think it's more useful to have a comment before the function.